README.md

```diff
@@ -700,3 +701,8 @@ This callback is if present is triggered when "clear all" button is clicked. You
 ### maxTags
 This prop specifies the maximum count of tags to be added. Incase the tags exceed, error will show up to convey the maximum tag limit has reached.
 
+### ariaAttrs
+
+An object used to pass custom accessibility attributes (ARIA properties) to enhance screen reader support and improve accessibility. For example, clearAllLabel defines the accessible label for a "clear all tags" button, providing context for assistive technologies.
```

Comment:
```suggestion
An object containing custom ARIA attributes to enhance screen reader support and improve accessibility. 
It accepts the below attributes
```js
{
  clearAllLabel?: string // Optional accessible label for a "clear all tags" button improving accessibility by providing a clear, descriptive action label.
}
```